src/rsch/raal/nrf_raal_api.h

```diff
@@ -47,133 +47,129 @@ extern "C" {
 
 /**
- * @brief Exit critical section of the module.
+ * @brief Function for exiting the critical section of the module.
  *
- * When this method is called driver has to expect the execution of the
- * @sa nrf_raal_timeslot_started or @sa nrf_raal_timeslot_ended
+ * When this method is called, the driver must expect the execution of the
+ * @sa nrf_raal_timeslot_started or the @sa nrf_raal_timeslot_ended
  * function.
  *
- * @note This function may be called when RAAL has already exited critical section. Ongoing call
- *       may NOT be interrupted by another call from IRQ with higher priority.
+ * @note This function can be called when RAAL has already exited the critical section. The ongoing call
+ *       cannot be interrupted by another call from an IRQ with higher priority.
  */
 void nrf_raal_critical_section_exit(void);
 
 /**
- * @brief RAAL client calls this method to notify radio driver about the start of a timeslot.
+ * @brief Function for notifying the radio driver about the start of a timeslot. Called by the RAAL client.
  *
- * The radio driver now has exclusive access to the peripherals until @p nrf_raal_timeslot_ended
+ * The radio driver has now exclusive access to the peripherals until @p nrf_raal_timeslot_ended
  * is called.
  *
  * @note The high frequency clock must be enabled when this function is called.
- * @note The end of the timeslot will be indicated by @p nrf_raal_timeslot_ended function.
+ * @note The end of the timeslot is indicated by the @p nrf_raal_timeslot_ended function.
  *
  */
 extern void nrf_raal_timeslot_started(void);
 
 /**
- * @brief RAAL client calls this method to notify radio driver about the end of a timeslot.
+ * @brief Function for notifying the radio driver about the end of a timeslot. Called by the RAAL client.
  *
- * Depending on the RAAL client configuration, radio driver has NRF_RAAL_MAX_CLEAN_UP_TIME_US
- * microseconds to do any clean-up actions on RADIO peripheral and stop using it completely.
- * Thus arbiter has to call this function NRF_RAAL_MAX_CLEAN_UP_TIME_US microseconds before
- * timeslot is finished.
+ * Depending on the RAAL client configuration, the radio driver has NRF_RAAL_MAX_CLEAN_UP_TIME_US
+ * microseconds to do any clean-up actions on the RADIO peripheral and stop using it.
+ * For this reason, the arbiter must call this function NRF_RAAL_MAX_CLEAN_UP_TIME_US microseconds
+ * before timeslot is finished.
  *
- * If RAAL is in the continuous mode, the next timeslot will be indicated again by
- * @p nrf_raal_timeslot_started function.
+ * If RAAL is in the continuous mode, the next timeslot is indicated again by
+ * the @p nrf_raal_timeslot_started function.
  *
- * This method shall not be called if @p nrf_raal_continuous_mode_exit has been called. Radio
- * driver shall assume that timeslot has been finished after @p nrf_raal_continuous_mode_exit
+ * This method is not to be called if @p nrf_raal_continuous_mode_exit has been called. The radio
```

Comment:
This paragraph is obsolete.
```suggestion
 * This function is called also when @p nrf_raal_continuous_mode_exit has been called while the timeslot has been granted.
```

Reply:
You mean the whole paragraph or just the single line? Asking because you edited out "The radio" at the end of the line.

Reply:
I meant the full paragraph, sorry for the late response. We can fix it in the 2nd batch